diff --git a/node_modules/@aztec/circuits.js/dest/structs/client_ivc_proof.js b/node_modules/@aztec/circuits.js/dest/structs/client_ivc_proof.js
index 813f31d..542e895 100644
--- a/node_modules/@aztec/circuits.js/dest/structs/client_ivc_proof.js
+++ b/node_modules/@aztec/circuits.js/dest/structs/client_ivc_proof.js
@@ -1,5 +1,5 @@
 import { BufferReader, serializeToBuffer } from '@aztec/foundation/serialize';
-import * as fs from 'fs/promises';
+// import * as fs from 'fs/promises';
 import path from 'path';
 /**
  * TODO(https://github.com/AztecProtocol/aztec-packages/issues/7370) refactory this to
